fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -785,6 +785,11 @@ fun homeScreen(interact: HomeScreenRobot.() -> Unit): HomeScreenRobot.Transition
     return HomeScreenRobot.Transition()
 }
 
+fun homeScreenWithCompose(composeTestRule: HomeActivityComposeTestRule, interact: ComposeTopSitesRobot.() -> Unit): ComposeTopSitesRobot.Transition {
```

Comment:
Let's rename it to `homeScreenWithComposeTopSites` ? (the most part of the home screen is already refactored to compose)